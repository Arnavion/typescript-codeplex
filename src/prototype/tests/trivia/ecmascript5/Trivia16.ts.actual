[
    {
        "kind": "SemicolonToken",
        "width": 1,
        "fullWidth": 5,
        "text": ";",
        "hasTrailingTrivia": true,
        "hasTrailingComment": true,
        "trailingTrivia": [
            {
                "kind": "WhitespaceTrivia",
                "text": " "
            },
            {
                "kind": "MultiLineCommentTrivia",
                "text": "/*\r"
            }
        ]
    },
    {
        "kind": "EndOfFileToken",
        "width": 0,
        "text": ""
    }
]