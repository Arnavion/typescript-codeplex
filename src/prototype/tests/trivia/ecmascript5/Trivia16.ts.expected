[
    {
        "kind": "SemicolonToken",
        "width": 1,
        "fullWidth": 17,
        "text": ";",
        "hasTrailingTrivia": true,
        "hasTrailingCommentTrivia": true,
        "hasTrailingNewLineTrivia": true,
        "trailingTrivia": [
            {
                "kind": "WhitespaceTrivia",
                "text": " "
            },
            {
                "kind": "MultiLineCommentTrivia",
                "text": "/*\r\n*/"
            },
            {
                "kind": "WhitespaceTrivia",
                "text": " "
            },
            {
                "kind": "MultiLineCommentTrivia",
                "text": "/*\r\n*/"
            },
            {
                "kind": "NewLineTrivia",
                "text": "\r\n"
            }
        ]
    },
    {
        "kind": "EndOfFileToken",
        "width": 0,
        "text": ""
    }
]